integration-tests/flyway/src/main/java/io/quarkus/it/flyway/FlywayFunctionalityResource.java

```diff
@@ -27,4 +27,5 @@ public String doMigrateAuto() {
                 "Version is null! Migration was not applied");
         return version.toString();
     }
+
```

Comment:
Is this line break necessary?

Reply:
I'm using [Quarkus Code Style](https://github.com/quarkusio/quarkus/blob/master/CONTRIBUTING.md) and before Commit, I run "mvn clean install" comands to format the code corretly